NitroxServer/Communication/Packets/PacketHandler.cs

```diff
@@ -67,27 +44,35 @@ public void Process(Packet packet, Connection connection)
         
         private void ProcessAuthenticated(Packet packet, Player player)
         {
-            PacketProcessor packetProcessor;
-            if (authenticatedPacketProcessorsByType.TryGetValue(packet.GetType(), out packetProcessor))
+            try
             {
-                packetProcessor.ProcessPacket(packet, player);
+                Type serverPacketProcessorType = typeof(AuthenticatedPacketProcessor<>);
+                Type packetType = packet.GetType();
+                Type packetProcessorType = serverPacketProcessorType.MakeGenericType(packetType);
+
+                PacketProcessor processor = (PacketProcessor)NitroxServiceLocator.LocateService(packetProcessorType);
+                processor.ProcessPacket(packet, player);
             }
-            else
+            catch
```

Comment:
Will this create an excessive amount of caught exceptions?  Many packets use the default packet processor.  What is the performance impact?